app/javascript/packs/utils.js

```diff
@@ -1,12 +1,32 @@
 import Typed from 'typed.js'
 
 document.addEventListener("turbolinks:load", function() {
-  const el = document.querySelector('.typed');
+  const el = document.querySelector('.typed');  
 
   if (el){
     new Typed('.typed', {
       strings: ["Fast","^1500 Open","^1500 Advanced", "^1500 Frictionless"],
       typeSpeed: 80
     });
   }
-})
\ No newline at end of file
+
+  if ($(".twitter-timeline")) {
+    // Twitter Overrides -----------------------------------------------------------------------------
+    $('.twitter-feed').delegate('iframe[data-widget-id="profile:opeNodeio"]', 'DOMSubtreeModified propertychange', function () {
+      $('.twitter-feed').find('.twitter-timeline').contents().find('.timeline-Tweet-media').css('display', 'none');
+      $('.twitter-feed').find('.twitter-timeline').contents().find('span.TweetAuthor-screenName').css('font-size', '16px');
+      $('.twitter-feed').find('.twitter-timeline').contents().find('p.timeline-tweet-text').css('font-size', '20px');
+      $('.twitter-feed').find('.twitter-timeline').contents().find('p.timeline-tweet-text').css('line-height', '1.6');
+      $('.twitter-feed').find('.twitter-timeline').contents().find('.timeline-Tweet').css({'border-bottom':'1px solid #252829','padding-bottom':'20px','margin-bottom':'20px'});
```

Comment:
don't use more than 95 columns

Reply:
gotcha - keeping this in mind for the future

Reply:
sometimes it's a little difficult when using a couple of extra bootstrap classes to style some elements, but I'll do my best to keep them short.